hack/generator/pkg/codegen/testdata/EmbeddedTypes/Embedded_type_simple_resource.golden

```diff
@@ -78,6 +81,34 @@ func (fakeResourceSpecArm FakeResource_SpecArm) GetType() string {
 	return string(fakeResourceSpecArm.Type)
 }
 
+type EmbeddedTestType struct {
+}
+
+var _ genruntime.ArmTransformer = &EmbeddedTestType{}
+
+// ConvertToArm converts from a Kubernetes CRD object to an ARM object
+func (embeddedTestType *EmbeddedTestType) ConvertToArm(name string) (interface{}, error) {
+	if embeddedTestType == nil {
+		return nil, nil
+	}
+	var result EmbeddedTestTypeArm
+	return result, nil
```

Comment:
Shouldn't this method include an assignment or two to copy data across? This does nothing.

Reply:
Or is this related to the missing property that I noted earlier?

Reply:
Yes, `EmbeddedTestType` was empty before so there was nothing to copy.

Reply:
Really good catch here